src/pages/dashboard.tsx

```diff
@@ -37,7 +45,7 @@ export function TableHero() {
 
   const tableHeaders = userColumns.map(column =>
     <TableHeaderCell
-      className="text-gray-300 py-5 border-b border-gray-300 cursor-pointer capitalize"
+      className="text-white hover:text-secondary py-5 border-b border-gray-300 cursor-pointer capitalize"
```

Comment:
please see the patterns, this should be something like text-dark

the "dark" class is for a default behaviour in dark mode, check out the hero and some other places.

tbh, just removing the class should be enough because the body has text-dark as a default, let's move away from in-line classes as much as possible, if it's not required.